datalake/src/main/java/com/sequenceiq/datalake/configuration/AppConfig.java

```diff
@@ -51,6 +57,15 @@
     @Named("freeIpaServerUrl")
     private String freeIpaServerUrl;
 
+    @Value("${datalake.intermediate.threadpool.core.size}")
```

Comment:
Why is this needed?

Reply:
It is necessary because of the added cloud-xxx dependencies.